### PR TITLE
Remove unused css and move some more stuff to tailwind

### PR DESCRIPTION
**Pull request by Tnixc**
- should look the same
- totally removes old variables
- after this we can start working on new new ui properly
- removed IconButton in favor of Button with icon size variant
- Added twMerge to button

### CODE REVIEW

**Review by hrfarmer — APPROVED** [review]
lgtm

**Comment by brw** [comment]
nice branch name spelling :pepeLaugh:
feel free to merge after review btw @duduBTW

**Comment by Tnixc** [comment]
unironically did not no refactor does not have an r there

**Comment by brw** [comment]
No need to keep merging master into this if there are no merge conflicts btw.
@duduBTW can this be merged yet?

**Comment by Tnixc** [comment]
got it. might be more effective to ping in discord but it's not urgent

File: src/renderer/src/components/icon-button/IconButton.tsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import "./styles.css";
 import { Component, JSX } from "solid-js";
 
 type Props = JSX.IntrinsicElements["button"];
 const IconButton: Component<Props> = (props) => {
-  return <button {...props} class={`icon-button ${props.class}`} />;
+  return (
+    <button
+      {...props}
+      class={`-m-2 flex h-6 w-6 items-center justify-center rounded-full p-2 text-2xl hover:bg-surface focus:bg-surface ${props.class}`}
```
Comment (duduBTW) [inline src/renderer/src/components/icon-button/IconButton.tsx:8]: You can use `cn` here.

Comment (Tnixc) [reply src/renderer/src/components/icon-button/IconButton.tsx:8]: I think I'm just gonna remove icon button entirely

Comment (duduBTW) [reply src/renderer/src/components/icon-button/IconButton.tsx:8]: Add is a a variant of the button component instead?

Comment (Tnixc) [reply src/renderer/src/components/icon-button/IconButton.tsx:8]: Yeah

File: src/renderer/src/components/button/Button.tsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ const buttonStyles = cva(
       size: {
         medium: "px-4 py-2",
         large: "px-7 py-2.5",
+        icon: "grid place-items-center aspect-square size-10 p-1",
```
Comment (duduBTW) [inline src/renderer/src/components/button/Button.tsx:19]: Add a negative negative margin to compensate for the white spacing created by the size/padding (`-m-3`), that way the icons are properly aligned with the content.

With `-m-3`
image

Without it:
image

**Review by duduBTW — APPROVED** [review]
lgtm

**Pull request merged**